### PR TITLE
The changes referenced on the Projectboard

### PR DESCRIPTION
**Pull request by Simaris**
Referenced Issues: #3 , #11 , #9 , #2 , #8 
To #8 
  Now, the browser path of each rendered uri might look weird:
    the file for \<http//:www.dkonrad-zuse-akademie-hoyerswerda.de#id\>
    resides in /rdfsites/http/www_dkonrad-zuse-akademie-hoyerswerda_dde#id/index.html
      which might be impractical to use unless we provide some sort of parser to easily find the rendition
      of each resource

### CODE REVIEW

**Comment by white-gecko** [comment]
Is `<http//:www.dkonrad-zuse-akademie-hoyerswerda.de#id>` a typo or should it be `<http://www.konrad-zuse-akademie-hoyerswerda.de/#id>` resp. `<http://www.konrad-zuse-akademie-hoyerswerda.de#id>`?

Why is thre resulting directory `/rdfsites/http/www_dkonrad-zuse-akademie-hoyerswerda_dde#id/index.html` ? shouldn't it be `/rdfsites/http/www.konrad-zuse-akademie-hoyerswerda.de/index.html`, while the index.html can optionally contain anchors?

**Comment by Simaris** [comment]
it was a typo,
The previous version of jekyll-rdf used dots as navigation points (it would create .../www/konrad...)
so I assumed that the previous group found some incompatibilities with some OS folder naming system. Thats why I decided to encode "." with "_d".

**Comment by white-gecko** [comment]
ah, ok so the `d` was intended. Could you please research, if there are systems, where a dot causes problems in directory names?

**Comment by Simaris** [comment]
none, it seems like a false assumption

**Comment by white-gecko** [comment]
which behavior, and why would it be hard?

**Comment by Simaris** [comment]
*wouldn't, a typo again
I changed its behaviour so that it now places the rendition at /rdfsites/http/www.konrad-zuse-akademie-hoyerswerda.de/index.html

File: lib/jekyll/drops/rdf_resource_class.rb [hunk 1]
```diff
@@ -35,14 +35,20 @@ class RdfResourceClass < RdfResource
       @subClassHierarchyValue = 0
       attr_accessor :lock
       attr_accessor :template
+      attr_accessor :secondaryTemplates
       attr_accessor :subClasses
```
Comment (white-gecko) [inline lib/jekyll/drops/rdf_resource_class.rb]: This could maybe be `alternativeTemplates`

**Pull request merged**